src/data/checkfiles/default_checkfiles/unused-metadata.js

```diff
@@ -0,0 +1,148 @@
+import { Link } from "@reach/router";
+import React from "react";
+import axios from "axios";
+import { Chip } from '@material-ui/core';
+
+export default {
+  name: "All metadata is used",
+  description:
+    "Checks if every piece of metadata is associated with a user, resource, data object, or collection.",
+  minimum_server_version: "4.2.0",
+  maximum_server_version: "",
+  interval_in_seconds: 3600,
+  active: true,
+  checker: async function () {
+    let result = {
+      status: "",
+      message: "",
+      success: 0,
+      failed: [],
+    };
+
+    let totalMeta = new Set(); // represents all metadata in the system (not just the ones that are used)
+    let numUnusedMeta = 0; // represents the number of metadata that are not used (not associated with a user, resource, data object, or collection)
+
+    let specificQuery =
+      "SELECT count(t.*) FROM (select meta_id from R_META_MAIN except select meta_id from R_OBJT_METAMAP) t;";
```

Comment:
consider using better variable name...  't' -> 'unused'

and I don't think you need the trailing semicolon